inheritance/src/main/java/inheritance/Review.java

```diff
@@ -0,0 +1,27 @@
+package inheritance;
+
+public class Review {
+    public String body;
+    public String author;
+    public double starsRating;
+    public String restaurantName;

```

Comment:
This works, but instead of holding a String restaurantName; You could have: public Restaurant restaurant; which would hold a reference to the object the Review is about.